lib/portage/dbapi/bintree.py

```diff
@@ -1234,7 +1234,16 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER")
+        ret = os.system(portage_trust_helper)
```

Comment:
Try `subprocess.run`?

Reply:
getuto is a shell script, so (I suspect) we'd need to set `shell=True` ?

We could capture the output into a log file (do we want to?)

Reply:
Added a comment that os.system is fine here.

Reply:
ok